gridcapa-core-valid-api/src/main/java/com/farao_community/farao/core_valid/api/resource/CoreValidResponse.java

```diff
@@ -23,14 +23,16 @@
 
     @Id
     private final String id;
-    private final String resultFileUrl;
+    private final String mainResultFileUrl;
+    private final String rexResultFileUrl; // TODO: remove this once choice of result file type can be done in the configuration
     private Instant computationStartInstant;
     private Instant computationEndInstant;
```

Comment:
You should use final if the variable wont change

Reply:
I agree. we'll do that in an other PR, I merge this one